src/slate-editor/slate-serialization.stories.js

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import { SlateContainer } from "../slate-container/slate-container";
+import { textToSlate } from "../common/slate-types";
+import { serializeValue } from "./serialization";
+import "./slate-serialization.stories.scss";
+
+export default {
+  title: "Serialization"
+};
+
+const serializationText = "This example shows the serialized editor content.";
+
+export const Serialization = () => {
+  const [value, setValue] = useState(serializationText);
+  const slateValue = textToSlate(serializationText);
+  const serializedValue = serializeValue(slateValue.toJSON());
```

Comment:
When I saw how much trouble you went through to serialize the value, I was going to ask you in my review why you weren't just calling the existing `serializeValue()` function -- and then I realized that the reason was that I had failed to write one. 😜 So in my commit I went ahead and wrote the missing `serializeValue()` function.

Reply:
Sounds good!